docs/content/execute.md

```diff
@@ -161,3 +213,86 @@ tags: [raises-exception]
 ---
 print(thisvariabledoesntexist)
 ```
+
+### Dealing with code that produces stderr
+
+You may also wish to control how stderr outputs are dealt with.
+
+
+Alternatively, you can configure how stdout is dealt with at a global configuration level, using the `nb_output_stderr` configuration value.
+This can be set to:
+
+You can configure the default behaviour for all notebooks in your `_config.yml`:
+
+```yaml
+execute:
+  stderr_output: show
+```
+
+Where the value is one of:
+
+* `"show"` (default): show all stderr (unless a `remove-stderr` tag is present)
+* `"remove"`: remove all stderr
+* `"remove-warn"`: remove all stderr, but log a warning if any found
+* `"warn"`, `"error"` or `"severe"`: log the stderr at a certain level, if any found.
+
+You can also remove stderr at a cell level, sing the `remove-stderr` [cell tag](https://jupyter-notebook.readthedocs.io/en/stable/changelog.html#cell-tags), like so:
+
+````md
+```{code-cell} ipython3
+:tags: [remove-stderr]
+
+import sys
+print("this is some stdout")
+print("this is some stderr", file=sys.stderr)
+```
+````
+
+```{code-cell} ipython3
+:tags: [remove-stderr]
+
+import sys
+print("this is some stdout")
+print("this is some stderr", file=sys.stderr)
+```
+
+### Dealing with code that produces stdout
+
+Similar to stderr, you can remove stdout at a cell level with the `remove-stdout` tag:
+
+````md
+```{code-cell} ipython3
+:tags: [remove-stdout]
+
+import sys
+print("this is some stdout")
+print("this is some stderr", file=sys.stderr)
+```
+````
+
+```{code-cell} ipython3
+:tags: [remove-stdout]
+
+import sys
+print("this is some stdout")
+print("this is some stderr", file=sys.stderr)
+```
+
+## Execution statistics
+
+As notebooks are executed, certain statistics are stored on the build environment by MyST-NB.
+The simplest way to access and visualise this data is using the `nb-exec-table` directive.
```

Comment:
```suggestion
The simplest way to access and visualise this data is using the **`{nb-exec-table}`** directive.
```

I saw you did this elsewhere and kinda liked it...maybe we want to adopt that as a standard way to refer to directives in these docs?

Reply:
sounds reasonable; you could even have a role that standardises the format, e.g. ``{myst-role}`nb-exec-table` ``